geometric/tests/addons.py

```diff
@@ -63,6 +69,12 @@ def workqueue_found():
     _plugin_import("openmm") is False and _plugin_import("simtk.openmm") is False, reason="could not find openmm. please install the package to enable tests")
 using_workqueue = pytest.mark.skipif(
     (not workqueue_found()), reason="could not find work_queue module or work_queue_worker executable. please install the package to enable tests")
+using_bigchem_psi4 = pytest.mark.skipif(
+    (not bigchem_found("psi4")), reason="BigChem/Psi4 is not working. please install the package to enable tests")
+using_bigchem_qchem = pytest.mark.skipif(
+    (not bigchem_found("qchem")), reason="BigChem/QChem is not working. please install the package to enable tests")
+using_bigchem_terachem = pytest.mark.skipif(
+    (not bigchem_found("terachem")), reason="BigChem/TeraChem is not working. please install the package to enable tests")
```

Comment:
Update to use the `bigchem_available` function I gave you, or else you will be executing actual calculations for these checks, which you do not want!